nemo/collections/nlp/nm/losses/masked_xentropy_loss.py

```diff
@@ -42,16 +42,16 @@
 from nemo.core.neural_types import LabelsType, LengthsType, LogitsType, LossType, NeuralType
 from nemo.utils.decorators import add_port_docs
 
-__all__ = ['TRADEMaskedCrossEntropy', 'CrossEntropyLoss3D']
+__all__ = ['MaskedXEntropyLoss']
 
 
-class TRADEMaskedCrossEntropy(LossNM):
+class MaskedXEntropyLoss(LossNM):
```

Comment:
Can we consistent in using either XEntropy or CrossEntropy? I vote for CrossEntropy

Reply:
Changed it to MaskedLogLoss.

Reply:
why not MaskedCrossEntropyLoss?